elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -3,36 +3,81 @@
+        <div class="form-check {{#inline}}form-check-inline d-inline-flex{{/inline}}{{^inline}}d-flex{{/inline}} align-items-center py-1">
+            <input class="form-check-input mt-0" type="radio"
+                name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
+                {{#checked}}checked{{/checked}} id="{{name}}-{{key}}">
+
+            <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
+                {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
+                <div class="ml-1 mr-1">{{{html}}}</div>
+            </label>
+
+            {{#display_feedback}}
+            <div class="pl-multiple-choice-feedback">
+                {{{feedback}}}
+            </div>
+            {{/display_feedback}}
+
+            {{#display_score_badge}}
+                {{#correct}}
+                    <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
+                {{/correct}}
+                {{#incorrect}}
+                    <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
+                {{/incorrect}}
+                {{#partial}}
+                    <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
+                {{/partial}}
+            {{/display_score_badge}}
         </div>
-        {{/display_feedback}}
-
-        {{#display_score_badge}}
-            {{#correct}}
-                <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
-            {{/correct}}
-            {{#incorrect}}
-                <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
-            {{/incorrect}}
-            {{#partial}}
-                <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
-            {{/partial}}
-        {{/display_score_badge}}
+    {{/answers}}
+{{/radio}}
+{{^radio}}
+<script>
+    $(function(){
+        $(document).ready(function(){
+            $(".pl-dropdown-choice-{{uuid}}.active").each(function(){
+                $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            });
+        });
+
+        $(".pl-dropdown-choice-{{uuid}}").click(function(){
+            // First, set dropdown button to display same text as selected button
+            $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            
+            // Next, change active choice to be what the user just selected
+            $(".pl-dropdown-choice-{{uuid}}").each(function(){
+                $(this).removeClass("active");
+            });
+            $(this).addClass("active");
+
+            // Finally, set input value
+            $("#pl-dropdown-input-{{uuid}}").val($(this).attr("value"));
+        });
+    });
+</script>
+<div class="dropdown py-1 d-inline-flex">
```

Comment:
```suggestion
<div class="dropdown py-1 d-inline-flex" role="listbox">
```
I'm not an expert on accessibility, but I believe this is necessary for screen readers. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role#examples. Also add `role="option"` to the options.

Reply:
I have removed it from the latest version, as `select` already implicitly has this role.